app/admin/todo_list.rb

```diff
@@ -4,7 +4,7 @@
   # See permitted parameters documentation:
   # https://github.com/activeadmin/activeadmin/blob/master/docs/2-resource-customization.md#setting-up-strong-parameters
   #
-  permit_params :title, :deadline,
+  permit_params :title, :deadline, {:user_ids => []},
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.